projects/llm-semconv.md

```diff
@@ -0,0 +1,82 @@
+# LLM Observability Semantic Conventions
+
+## Description
+
+This project aims to establish observability standards for Large Language Model (LLM) applications, focusing on API interactions and multi-step LLM frameworks. Key areas include:
+
+- Developing semantic conventions for standard and complex LLM interactions.
+- Initiating vendor-specific conventions as a foundation for broader contributions.
+- Concentrating on attributes relevant to Metrics and Spans.
+
+Example Use Cases:
+
+- **Debugging**: Standardizing log data and error reporting for efficient error resolution in LLM applications.
+- **Evaluation**: Collecting signal for evaluation of LLM performance, including response accuracy, context retention, and user feedback.
+- **Cost Tracking**: Monitoring operational costs like computation time and API calls for resource optimization.
+- **AI Safety**: Monitoring for ethical LLM use, including privacy compliance, bias detection, and secure data handling.
+
+For Spans: Instrumentation and debugging guidance for multi-step LLM applications, with recommendations on user privacy and intellectual property security, marking sensitive attributes and events as opt-in.
+
+For Metrics: Tracking model performance by collecting and correlating data like model names, versions, parameters, latency, and evaluation scores.
+
+## Deliverables
+
+- Experimental semantic conventions for LLMs, allowing flexibility for evolution in this fast-paced field.
+- Reference prototypes demonstrating the application of these conventions in data collection, processing, and analysis.
+- Guidelines for monitoring LLM applications, including implementation strategies and privacy/security considerations.
+
+## Staffing / Help Wanted
+
+We will gather further staff as the project is established. We need to 
+make sure there is enough representation across both vendors and customers
+who will benefit from these semantic conventions.
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- @alolita
+- *Seeking additional sponsors*
+
+**Engineers:**
+
+- @nirga
+- @sudivate
+- @cartermp
+- *Positions open for more engineers*
+
+**Maintainers:**
+
+- To be determined
+
+## Meeting Times
+
+Alternating weekly meetings to accommodate different time zones:
+
+- Week 1: Thursdays at 8 AM PST
```

Comment:
I can't make Thu 8am work. 

Could we add this poll and try to find the most suitable time?

```suggestion
https://strawpoll.com/3RnYlN4Jxye
```